multicluster/apis/multicluster/constants/constants.go

```diff
@@ -24,7 +24,8 @@ const (
 	ServiceImportKind              = "ServiceImport"
 	ClusterInfoKind                = "ClusterInfo"
 
-	ResourceExportFinalizer = "resourceexport.finalizers.antrea.io"
+	LegacyResourceExportFinalizer          = "resourceexport.finalizers.antrea.io"
+	DomainQualifiedResourceExportFinalizer = "finalizers.antrea.io/resourceexport"
```

Comment:
I am not sure we need the `DomainQualified` prefix here, it could just be the new `ResourceExportFinalizer`.

The K8s documentation you linked to (https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#finalizers) suggests that a better name may be `resourceexport.antrea.io/finalizer`, or maybe `mc.antrea.io/resourceexportfinalizer`

Reply:
Done